packages/app-desktop/gui/NoteEditor/utils/contextMenuUtils.test.ts

```diff
@@ -0,0 +1,35 @@
+import { svgDimensions } from './contextMenuUtils';
+
+describe('contextMenuUtils',()=>{
+	test('should return the correct dimensions for svg with space seperated viewBox values',()=>{
+		const svg = '<svg viewBox="0 0 45.9375 60" "></svg> ';
```

Comment:
```suggestion
		const svg = '<svg viewBox="0 0 45.9375 60"></svg> ';
```

Consider removing the extra `"`.